### ISSUE TITLE
[FireMonkey] Since v2.25, addValueChangeListener is broken

### ISSUE THREAD

**Issue by pintassilgo** [erosman/support#381]
```js
// ==UserScript==
// @name            test
// @match           http://*.example.com/
// @match           https://*.example.com/
// ==/UserScript==

GM.addValueChangeListener('test', () => {});
GM.setValue('test', Date());
```

Browser console:
> can't access property "oldValue", changes[id].storage is undefined

**Comment by pintassilgo** [comment]
btw, anchors in Help for `GM_getValue` and `GM_listValues` in `API Comparison` are broken.
You're using `#GM_getValue` but should use `#getValue`.

**Comment by erosman** [comment]
> can't access property "oldValue", changes[id].storage is undefined

I will fix that. Thank you.

> btw, anchors in Help for GM_getValue and GM_listValues in API Comparison are broken.
You're using #GM_getValue but should use #getValue.

I had already noticed that and fixed it gor the next release. :man_facepalming:

**Comment by erosman** [comment]
Thanks. It is due to changes in FM storage in 2.25 which were overlooked in `GM.addValueChangeListener` :man_facepalming: 

The `api.js` is also re-written considerably in v2.35.
I will fix it.

**Comment by erosman** [comment]
Fixed and will be in v2.35
